client/src/modules/exchange/exchange.modal.html

```diff
@@ -17,13 +17,14 @@
       <ui-select
         name="currency"
         ng-model="ModalCtrl.rate.currency"
+        ng-change="ModalCtrl.selectCurrency()"
         required>
         <ui-select-match placeholder="{{ 'FORM.PLACEHOLDERS.CURRENCY' | translate }}">
```

Comment:
Can you check to make sure this translation exists?  It doesn't look like it is rendering on my machine:

![image](https://user-images.githubusercontent.com/896472/114533138-7f754380-9c45-11eb-87fe-6e249a8082a8.png)

Reply:
Fixed